configure.ac

```diff
@@ -1283,6 +1283,107 @@ else
     PAC_APPEND_FLAG([-L${with_openpa_prefix}/lib],[WRAPPER_LDFLAGS])
 fi
 
+# Izem
+
+AC_ARG_ENABLE([izem],
+[  --enable-izem@<:@=ARGS@:>@   Enable features from the Izem library.
+                          "ARGS" is a list of comma separated features.
+                          Accepted arguments are:
+			  sync     - use the Izem interface for sychronization objects
+                                     (locks and condition variables) instead of the MPL interface
+                          yes/all  - all of the above features are enabled
+                          no/none  - none of the above features are enabled],,
+[enable_izem=no])
```

Comment:
Help messages should use `AS_HELP_STRING`.

Reply:
Is `izem` disabled by default?  Is it because of the thread oversubscription issue?

Reply:
yes and yes

Reply:
Not in this PR, but it'll be good to have an option to enable both MPL and izem and have an environment variable to pick one of them.  Reconfiguring mpich is too heavy weight for this.

Reply:
we could do that, but I would think for production build-time selection would better, since it would eliminate the runtime check and extra branch. The runtime selection would be useful for experimenting though

Reply:
Agreed.  For performance builds, a single option is better.